server/Controllers/changePicture.js

```diff
@@ -0,0 +1,44 @@
+import UserModel from "../Schemas/User";
+
+const createRandomId = (length) => {
+  let result = "";
+  const characters =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+};
+
+const changePicture = async (req, res) => {
+  if (!req.files) {
+    console.log("error files");
+  } else {
+    try {
+      const hashedPT = `${req.files.image.name.split(".")[0] +
+        createRandomId(5)}.${req.files.image.mimetype.split("/")[1]}`;
+      req.files.image.mv(`./server/data/avatar/${hashedPT}`, (e) => {
+        if (e) console.error(e);
+      });
```

Comment:
this code is the exact same code has in `server/Helpers/signUp.js` (line 49 to line 55).
Put it in a function in `server/Helpers/signUp.js` (`savePicture` for ex) that return `hashedPT` and use it in your code 👌🏻